doc/sphinx/Pacemaker_Explained/cluster-options.rst

```diff
@@ -588,16 +588,12 @@ values, by running the ``man pacemaker-schedulerd`` and
        If this is set to 0 (the default), the cluster will never assume
        watchdog-based self-fencing.
 
-       If this is set to a negative value, the cluster will use twice the local
-       value of the ``SBD_WATCHDOG_TIMEOUT`` environment variable if that is
-       positive, or otherwise treat this as 0.
+       Negative values are considered invalid and are being treated as 0.
```

Comment:
Change the type (above the description) from timeout to duration, and then you don't need this line

(The duration description should mention that the duration must be nonnegative, but that's not important for this PR)